src/app/modules/submission/submission.utils.ts

```diff
@@ -23,14 +23,10 @@ type ResponseModeFilterParam = {
 
 // Exported for testing.
 export const getResponseModeFilter = (
-  responseMode: FormResponseMode,
+  encrypted: boolean,
 ): (<T extends ResponseModeFilterParam>(responses: T[]) => T[]) => {
-  switch (responseMode) {
-    case FormResponseMode.Email:
-      return emailResponseModeFilter
-    case FormResponseMode.Encrypt:
-      return encryptResponseModeFilter
-  }
+  if (encrypted) return encryptResponseModeFilter
+  else return emailResponseModeFilter
```

Comment:
If you want to change the signature as such (encrypted vs non-encrypted), then would need to rename emailResponseModeFilter (since it is actually email + non-e2ee storage mode)

- Suggest add a todo, since this function would be deprecated once we deprecated e2ee storage mode

Reply:
Suggest add a todo also to deprecate this function once we deprecate e2ee storage mode

Reply:
Done in [4bf6286](https://github.com/opengovsg/FormSG/pull/6595/commits/4bf6286022cba46dd59163f21e396c98fb21fbf5) and [7072960](https://github.com/opengovsg/FormSG/pull/6595/commits/7072960a6072c29aba71f49956e20a962735809b)!